### ISSUE TITLE
Color walk differently

### ISSUE THREAD

**Issue by theis999** [MortenTobiasNielsen/Factorio-TAS-Generator#246]
If it is the same X or same Y.

**Comment by MortenTobiasNielsen** [comment]
Good idea. It would be nice to know if it is a same X/Y, diagonal or a "normal" walk - I am though not really sure how to indicate it, given that all of the distinct color I have been able to find are already used.

**Comment by theis999** [comment]
I think we can use the same GUI space as intermediate walks if you out of reach. Since those can only be applied to non-walk tasks and this can only be applied to walk task. It would also make sense since it is effectively the same type of information. Then we would only need distinct colors between these.

We would need to make sure that intermediate walk steps don't color using the same X / same Y. But that should be manageable.

**Comment by MortenTobiasNielsen** [comment]
That sounds like a way to do it, but I would probably need to see it. :)
